src/content/docs/bff/architecture/multi-frontend.md

```diff
@@ -60,6 +60,41 @@ After your application's logic is executed, there are two middlewares registered
 
 5. `MapRemoteRoutesMiddleware` - This will handle any configured remote routes. Note, it will not handle plain YARP calls, only routes that are specifically added to a frontend.
     
-6. `ProxyIndexMiddleware` - If configured, this proxy the `index.html` to start the browser based app.  
+6. `ProxyIndexMiddleware` - If configured, this proxies the `index.html` to start the browser based app.  
 
+If you don't want this automatic mapping of BFF middleware, you can turn it off using `BffOptions.AutomaticallyRegisterBffMiddleware`. When doing so, you'll need to manually register and add the middlewares:
+
+```csharp
+var app = builder.Build();
+
+app.UseBffFrontendSelection();
+app.UseBffPathMapping();
+app.UseBffOpenIdCallbacks();
+
+// TODO: your custom middleware goes here
+app.UseRouting(); 
+app.UseBff();
+
+// NOTE: Only add this if you want to proxy remote APIs. 
+app.UseBffRemoteRoutes();
+
+app.MapBffManagementEndpoints();
+app.UseBffIndexPages();
+
+app.Run();
+```
+
+## Authentication Architecture
+
+When you use multiple frontends, you can't rely on [manual authentication configuration](../fundamentals/session/handlers.mdx#manually-configuring-authentication). This is because each frontend requires its own scheme, and potentially it's own OpenID Connect and Cookie configuration. 
```

Comment:
```suggestion
When you use multiple frontends, you can't rely on [manual authentication configuration](../fundamentals/session/handlers.mdx#manually-configuring-authentication). This is because each frontend requires its own scheme, and potentially its own OpenID Connect and Cookie configuration. 
```